src/main/java/uk/ac/ebi/intact/app/internal/model/filters/edge/EdgeNegativeFilter.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ebi.intact.app.internal.model.filters.edge;
+
+import uk.ac.ebi.intact.app.internal.model.core.elements.edges.Edge;
+import uk.ac.ebi.intact.app.internal.model.core.view.NetworkView;
+import uk.ac.ebi.intact.app.internal.model.filters.BooleanFilter;
+
+public class EdgeNegativeFilter extends BooleanFilter<Edge> {
```

Comment:
As mentioned, it would be better to have a single filter with 3 options (positive only, positive and negativ, negative only), as we do on the web.